pkgs/top-level/all-packages.nix

```
@@ -34201,4 +34202,24 @@ with pkgs;
   };
 
   zthrottle = callPackage ../tools/misc/zthrottle { };
+
+  # Only expose images for mono-platform stdenv's
+  #
+  # Conditional logic must use super, as using pkgs will
+  # cause infinite recursion when trying to determine a fixed point
+
+} // lib.optionalAttrs (super.stdenv.isLinux && (super.hostPlatform.system == super.targetPlatform.system)) {
```

```suggestion
} // lib.optionalAttrs (super.stdenv.isLinux && super.hostPlatform.system == super.targetPlatform.system) {
```
Technically unnecessary, just liked how they read.